### PR TITLE
Host monitor mini introduction

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

File: conf/hmm/local.conf.sample [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    HALT,${TMPDIR},100M,1K \
+    HALT,${DL_DIR},100M,1K \
+    HALT,${SSTATE_DIR},100M,1K \
+    HALT,/tmp,10M,1K"
+
+PACKAGECONFIG:append:pn-qemu-system-native = " sdl"
+PACKAGECONFIG:append:pn-nativesdk-qemu = " sdl"
+
+CONF_VERSION = "2"
+ACCEPT_FSL_EULA = "1"
```
Comment (albsod) [inline conf/hmm/local.conf.sample:23]: IMX specific?

File: conf/hmm/local.conf.sample [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+INHERIT += "rm_work"
+
+DISTRO_FEATURES:append = " systemd"
+DISTRO_FEATURES:append = " virtualization"
+DISTRO_FEATURES:append = " kvm"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
```
Comment (albsod) [inline conf/hmm/local.conf.sample:31]: Legacy?

File: conf/hmm/local.conf.sample (hunk 1 above)
Comment (RikardO-HM) [reply conf/hmm/local.conf.sample]: yes, fixed

File: conf/hmm/conf-notes.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Host Mobility targets are:
+    1. console-hostmobility-image *console only*
+    2. host-insight-image 'console only with host-insight-client support'
```
[the remark this answers is not in the document]
Comment (RikardO-HM) [reply conf/hmm/conf-notes.txt]: fixed

**Review by albsod — APPROVED** [review]
Nice work!

**Pull request merged**